backend/core/src/user/user.controller.ts

```diff
@@ -48,12 +49,19 @@ export class UserController {
   @Post()
   @UseGuards(OptionalAuthGuard, AuthzGuard)
   @ApiOperation({ summary: "Create user", operationId: "create" })
-  async create(@Body() dto: UserCreateDto): Promise<UserDtoWithAccessToken> {
+  async create(@Body() dto: UserCreateDto): Promise<UserDto> {
     const user = await this.userService.createUser(dto)
-    const accessToken = this.authService.generateAccessToken(user)
     // noinspection ES6MissingAwait
-    void this.emailService.welcome(user)
-    return mapTo(UserDtoWithAccessToken, { ...user, accessToken })
+    void this.emailService.welcome(user, dto.appUrl)
+    return mapTo(UserDto, user)
+  }
+
+  @Put("confirm")
+  @ApiOperation({ summary: "Forgot Password", operationId: "forgot-password" })
```

Comment:
The summary should be "Confirm email", operationId "confirm"

Reply:
Good catch